src/vcpkg/commands.setinstalled.cpp

```diff
@@ -36,6 +39,85 @@ namespace vcpkg::Commands::SetInstalled
         nullptr,
     };
 
+    static void create_dependency_graph_json(const ActionPlan& action_plan)
```

Comment:
This should be split into two functions:
- First, a testable function which returns the document as a Json object
- Second, an untestable function which uploads the document